specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -134,6 +142,14 @@
         },
         "description": "Restores all key materials using the SAS token pointing to a previously stored Azure Blob storage backup folder",
         "parameters": [
+          {
+            "name": "storageAuthType",
+            "in": "query",
+            "description": "Type of authentication to use",
```

Comment:
`description` is shown as part of the document for the REST API, isn't it? If so, we need to be more specific what we mean here, even though the parameter name might give a hint. 

We could say, for example: Specifies the authentication type that is used to authenticate Managed HSM with Azure Storage. The default option, `sasToken`, uses the provided SAS token for authentication/authorization. `managedIdentity` uses the user-assigned managed identity that was assigned to the managed HSM resource.